pubspec.yaml

```diff
@@ -14,12 +14,17 @@ dependencies:
     path: packages/presentation
 
   cupertino_icons: ^1.0.2
+  get_it: ^7.2.0
```

Comment:
same, u can get this dependency from domain -> presentation -> this modulethis
or u should delete this dependency from domain module

Reply:
removed it